src/components/Collection/Collection.stories.tsx

```diff
@@ -0,0 +1,510 @@
+import React from 'react'
+import { Collection } from './Collection'
+import { CollectionItem } from './CollectionItem'
+import { CollectionHeading } from './CollectionHeading'
+import { CollectionDescription } from './CollectionDescription'
+import { CollectionMeta } from './CollectionMeta'
+import { CollectionMetaItem } from './CollectionMetaItem'
+import { CollectionMetaItemTag } from './CollectionMetaItemTag'
+import { CollectionThumbnail } from './CollectionThumbnail'
+import { CollectionCalendarDate } from './CollectionCalendarDate'
+
+export default {
+  title: 'Components/Collection',
+  component: Collection,
+  parameters: {
+    docs: {
+      description: {
+        component: `
+### USWDS 2.10.0 Collection component
+        `,
+      },
+    },
+  },
+}
+
+export const collectionWithDefaultItems = (): React.ReactElement => (
+  <div className="tablet:grid-col-6">
+    <Collection>
+      <CollectionItem>
+        <CollectionHeading>
+          <a
+            className="usa-link"
```

Comment:
You can replace anchor tags with class `usa-link` with the `Link` component we have internally